src/universal_transfer_operator/data_providers/filesystem/__init__.py

```diff
@@ -28,7 +28,7 @@ def resolve_file_path_pattern(
     :param transfer_mode: Use transfer_mode TransferMode; native, non-native or thirdparty.
     """
     if transfer_params is None:
-        transfer_params = TransferParameters()
+        transfer_params = TransferIntegrationOptions()
```

Comment:
Should this still not be `TransferParameters()`?

Reply:
We need conn_id in transfer_params and those transfer_params when we pass them in the `get_transfer_integration` method.
ref : https://github.com/astronomer/apache-airflow-provider-transfers/blob/5a81abb8541b972164790e0e60234078750ce734/src/universal_transfer_operator/integrations/__init__.py#L12